tests/manager/api/test_sirsidynix_auth_provider.py

```diff
@@ -77,8 +77,13 @@ def headers(self, api: SirsiDynixHorizonAuthenticationProvider) -> dict[str, str
             "x-sirs-clientID": api.sirsi_client_id,
         }
 
-    def provider_mocked_api(self) -> MockedSirsiApi:
-        provider = self.provider()
+    def provider_mocked_api(
+        self,
+        provider: SirsiDynixHorizonAuthenticationProvider | None = None,
+        patron_status_info: dict | None = None,
```

Comment:
Minor: Would be good to fill in the generic info for the `dict` type hint here, probably `dict[str, Any]`.